internal/cmd/identities.go

```diff
@@ -247,6 +247,10 @@ func CreateIdentity(req *api.CreateIdentityRequest) (*api.CreateIdentityResponse
 
 	req.Kind = kind.String()
 
+	if kind == "machine" {
+		req.SetOneTimePassword = false
+	}
```

Comment:
This does not prevent the user from send a request for a machine user with `SetOneTimePassword=true`. Would it be better to have this check/mutation server-side?

Reply:
Made this change